pandas/core/base.py

```diff
@@ -603,9 +609,6 @@ def _aggregate_multiple_funcs(self, arg, _level, _axis):
                     keys.append(col)
                 except (TypeError, DataError):
                     pass
-                except ValueError:
```

Comment:
I am not sure what this is for just checked some tests locally and didn't fail

Reply:
Hmm a little wary here - this does show as hit on code coverage:

https://codecov.io/gh/pandas-dev/pandas/src/master/pandas/core/base.py#L608

Reply:
It seems this is for nuisance columns